### PR TITLE
Infra - use ui-autotools for ssr sanity tests

### PR DESCRIPTION
**Pull request by mosheKerbel**
@shlomitc 
@jonathanadler

### CODE REVIEW

File: src/Stepper/Stepper.meta.js [hunk 1]
```diff
@@ -2,7 +2,26 @@ import Stepper from './Stepper';
 import Registry from '@ui-autotools/registry';
 
 const metadata = Registry.getComponentMetadata(Stepper);
+metadata.nonEventListenerTestCompliant = true;
```
Comment (jonathanadler) [inline src/Stepper/Stepper.meta.js]: Why is this needed?

Comment (mosheKerbel) [reply src/Stepper/Stepper.meta.js]: this test is currently failing for react 15 becuase it uses Hydrate
need to modify the autotools code first.

Comment (mosheKerbel) [reply src/Stepper/Stepper.meta.js]: they have a good fallback for this in the SSR but in this test they don't

Comment (jonathanadler) [reply src/Stepper/Stepper.meta.js]: But isn't it `nonReactStrictModeCompliant` that fixes that?
I thought `nonEventListenerTestCompliant` is different, but I might be totally wrong.

**Comment by mosheKerbel** [comment]
merged.
failed due to 1 flaky unrelated timeout test.
